native/src/main/java/bblfsh/bash/Main.java

```diff
@@ -0,0 +1,18 @@
+package bblfsh.bash;
+
+public class Main {
+
+    public static void main(String args[]) {
+        final RequestReader reader = new RequestReader(System.in);
```

Comment:
You are right!

Please check out if I have used proper style when injecting the ObjectMapper.